nmosquery/common/routes.py

```diff
@@ -85,7 +85,12 @@ def __subscriptions_post(self):
             elif data["secure"] is False:
                 abort(400, "All subscriptions must be secure when operating with HTTPS")
         obj, created = self.query.post_ws_subscribers(data)
-        return (201 if created else 200, obj)
+        response = make_response(jsonify(obj), 201 if created else 200)
+        response.autocorrect_location_header = False
```

Comment:
Why are we choosing to use relative url's here? Apparently RFC 2616 says the location header should be an absolute path.

Reply:
The API typically runs behind a reverse proxy, and I'm not sure we have the relevant code in place to adjust these absolute paths on the fly (possibly a consideration for any new webapi...). This matches the behaviour in the Registration API.

Reply:
I believe RFC 2616 has been replaced by RFC7231, which allows both relative and absolute.
https://tools.ietf.org/html/rfc7231#section-7.1.2

Reply:
Check you out Jimmy Gibbs with ya RFC-foo. PWNED.